### PR TITLE
null safety is supported

### PR DESCRIPTION
**Pull request by II11II**

### CODE REVIEW

**Comment by dshukertjr** [comment]
This is great! I really hope this gets merged!!

File: tool/generate_fonts.dart [hunk 1]
```diff
@@ -22,7 +22,7 @@ void main(List<String> args) {
     "class FeatherIcons {\n"
   ];
 
-  for (Map<String, dynamic> icon in icons) {
+  for (Map<String, dynamic> icon in icons as Iterable<Map<String, dynamic>>) {
```
[the remark this answers is not in the document]
Comment (II11II) [reply tool/generate_fonts.dart:25]: Did you face problems?

[the remark this answers is not in the document]
Comment (bernaferrari) [reply tool/generate_fonts.dart:25]: No, but it is useless. It is like doing 12 as int.

[the remark this answers is not in the document]
Comment (II11II) [reply tool/generate_fonts.dart:25]: It helps to read code easily by the third party.
If there are no problems, it will be left as it is.

**Comment by dshukertjr** [comment]
Hi @NikSWE ! Thanks for creating this amazing package. 

Will a new version with null-safety be published to pub.dev soon?

**Comment by NikSWE** [comment]
Yes it will be published soon

**Pull request merged**